pkgs/development/tools/fsautocomplete/default.nix

```diff
@@ -0,0 +1,53 @@
+{ buildDotnetModule, mkNugetDeps, emptyFile, lib, dotnet-sdk }:
```

Comment:
Overall looks ok, however I would really prefer if the package was built from source instead of downloaded from nuget

Reply:
I've tried to do it, but it turned to be quite difficult. Unhelpfully, this package itself uses another tool - paket - for managing its nuget dependencies, so it's a bit a chicken and egg problem. I guess we can use this "binary" install approach as a stage-1 and later develop another method for building tools from sources.

Reply:
Fine, I personally had a couple problems with paket and Nix. We'll have to develop a solution to that, and then the package can be migrated to source build